.github/workflows/unit-test.yml

```diff
@@ -62,8 +62,9 @@ jobs:
           kubectl: 1.18.4
           kubeval: v0.16.1
           command: |
-            kubeval --strict ${{ env.K8S_MANIFEST_DIR }}/k8s/*.yml
-            kubeval --strict ${{ env.K8S_MANIFEST_DIR }}/k8s/kustomize/base/cloudbeat*.yml
+            kubeval --strict ${{ env.K8S_MANIFEST_DIR }}/k8s/*.yml --ignore-missing-schemas
```

Comment:
```suggestion
            kubeval --strict ${{ env.K8S_MANIFEST_DIR }}/k8s/*.yml
```

Reply:
The reason that I put the `--ignore-missing-schemas` is that `kubeval` cannot evaluate a non-k8s schema object. 

Therefore, in a case of a type such that `eksctl.io/v1alpha5` the `kubeval` will throw an error since it cannot evaluate it.

I would not want to limit the content in the directory to k8s only. Then we wouldn't able to use `kind` .yaml in this directory.

For example -> `kind.x-k8s.io/v1alpha4`;

+ I see that it does not check recursively I am fixing it.

Reply:
specifically for what I commented on, there's no need for it

Reply:
Yes, you are absolutely right, but I changed the code the first time you looked at it.
When you first review the code, it ran only on the parent directory(`deploy/k8s`) so no issues there, now it works recursively on all of the directories so without adding the flag it won't work.